pkg/reconciler/testing/table.go

```diff
@@ -85,18 +95,24 @@ func (r *TableRow) Test(t *testing.T, factory Factory) {
 		t.Errorf("Error capturing actions by verb: %q", err)
 	}
 
+	// Previous state is used to diff resource expected state for update requests that were missed.
+	objPrevState := map[string]runtime.Object{}
```

Comment:
I think we need to seed this with `r.Objects`

Reply:
Without this we will never practically see the text we want `Missing update ...` because we'll hit the never-created path, which are generally mutually exclusive.

Reply:
Done.